src/DipStrikeTool.hpp

```diff
@@ -60,7 +67,6 @@ class DipStrikeTool : public IVistaOpenGLDraw, public cs::core::tools::MultiPoin
   glm::dvec4 getInterpolatedPosBetweenTwoMarks(cs::core::tools::DeletableMark const& pMark1,
       cs::core::tools::DeletableMark const& pMark2, double value);
 
- private:
```

Comment:
Why has this been removed?

Reply:
At this point it is already a private block.